src/scripts/content/todoist.js

```diff
@@ -34,25 +35,68 @@ togglbutton.render('.task_item .content:not(.toggl)', {observe: true}, function
     return clone.textContent.trim();
   };
 
-  projectFunc = function () {
-    var projectElem, projectLabel;
-    projectElem = $('.project_link');
-    if (projectElem) {
-      return projectElem.textContent.trim();
+  getSidebarEle = function (elem) {
+    var editorInstance, projectId, sidebarRoot, sidebarEle;
+    editorInstance = elem.closest('.project_editor_instance');
```

Comment:
I think it doesn't work for me because I can't find this element appearing anywhere. Which view am I supposed to be in for this to work?